shell/platform/linux/testing/fl_test.cc

```diff
@@ -39,3 +48,14 @@ gchar* bytes_to_hex_string(GBytes* bytes) {
     g_string_append_printf(hex_string, "%02x", data[i]);
   return g_string_free(hex_string, FALSE);
 }
+
+FlEngine* make_mock_engine() {
```

Comment:
Can you split this change out into a separate PR?

Reply:
There's a number of cases where this could be common.

Reply:
Done, #21585.